tests/testData/mockData.ts

```diff
@@ -159,7 +159,7 @@ export const getMockRequestAndResponse = ({
         : '';
 
     cookieString += inputMethod
-        ? `${INPUT_METHOD_COOKIE}=%7B%22inputMethod%22%3A%22${inputMethod}%22%2C%22uuid%22%3A%22${inputMethodUuid}%22%7D;`
+        ? `${INPUT_METHOD_ATTRIBUTE}=%7B%22inputMethod%22%3A%22${inputMethod}%22%2C%22uuid%22%3A%22${inputMethodUuid}%22%7D;`
```

Comment:
this can be removed as we utilise the default session to you can INPUT_METHOD_ATTRIBUTE and any default values